src/pages/Country.module.scss

```diff
@@ -59,11 +60,16 @@
     margin-right: 70px;
   }
 
+  .btn {
+    margin-bottom: 0;
```

Comment:
I don't understand. You added marginBotton on line 40 for `.btn` and then you set it to 0. What is going on here?

Reply:
I just wanted to confuse the investigator :)